app/__init__.py

```diff
@@ -4,8 +4,14 @@
 from pathlib import Path
 import shutil
 import yaml
+import os
 
-BASE_PATH = Path(__file__).parents[1]
+APP_FOLDER = 'app'
+
+if os.getenv("BASE_PATH"):
+    BASE_PATH = Path(os.getenv("BASE_PATH"))
+else:
+    BASE_PATH = Path(__file__).parents[1]

```

Comment:
adding tasks that were PMed the other day:

- [ ] remove envvar and pass variables around instead (should we just move to a full blown config system like [confuse](https://github.com/beetbox/confuse) which would put all user configuration into a .config folder within their user's profile (and abstract all system level interactions for us... as any changes to config.yaml prompts for git changes which is annoying)

Reply:
I removed this dependency while also refactoring out references to enums and file paths from the main `__init__.py` where the server and boot up logic (aka explicit initialization logic is contained vs relying on implicit logic across several files).